src/components/Game.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import { play, weapons, Weapon, Result } from "../game/game";
+
+type State = {
+  winner: Result;
+  hasPlayed: boolean;
+};
+
+export default class Game extends React.Component<{}, State> {
+  state = {
+    winner: "Draw" as Result,
+    hasPlayed: false
+  };
+
+  play(playerWeapon: Weapon) {
+    const computerWeapon = weapons[Math.floor(Math.random() * weapons.length)];
```

Comment:
This should be removed from the incomplete version and `play()` should only take 1 param, having the candidate refactor it to make it more testable was very useful